packages/bull-vault/src/PayableBull.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity =0.7.6;
+
+// contract
+import {BullStrategy} from "./BullStrategy.sol";
+// lib
+import {StrategyMath} from "squeeth-monorepo/strategy/base/StrategyMath.sol"; // StrategyMath licensed under AGPL-3.0-only
+
+/**
+ * @notice PayableBull contract
+ * @author opyn team
+ */
+contract PayableBull {
+    using StrategyMath for uint256;
+
+    BullStrategy private bullStrategy;
+
+    constructor(address _bull) {
```

Comment:
We should be specific about var names here since bull tokens, bull contract, etc., but I can just make an additional PR that sets our variable names.